scripts/testnet/sovereignBridge/config/configs.cfg

```diff
@@ -8,6 +8,7 @@ WALLET="~/wallet.pem"
 WALLET_SOVEREIGN="~/MultiversX/testnet/node/config/walletKey.pem"
 
 #=========== NETWORK CONFIGURATION ===========
+MAIN_CHAIN_ELASTIC=https://testnet-index.multiversx.com
```

Comment:
Will this config be automatically updated to mainnet/devnet elastic link if we change the script configuration chain ID?